src/iidm/MergedBus.cpp

```diff
@@ -66,12 +67,11 @@ stdcxx::Reference<Component> MergedBus::getConnectedComponent() {
 unsigned long MergedBus::getConnectedTerminalCount() const {
     checkValidity();
 
-    unsigned long count = 0;
-    for (const auto& it : m_buses) {
-        count += it.get().getConnectedTerminalCount();
-    }
+    const auto& mapper = [](const std::reference_wrapper<ConfiguredBus>& bus) {
+        return bus.get().getConnectedTerminalCount();
+    };
 
-    return count;
+    return boost::accumulate(m_buses | boost::adaptors::transformed(mapper), 0);
```

Comment:
To be checked: is it more efficient written like this?

Reply:
Well, in release mode, execution time of this function are stricly the same (0.0003 ms :/)
In debug mode, it is slower
=> I will revert